src/Common/Thumbnail/Thumbnail.scss

```diff
@@ -77,9 +77,11 @@
     }
   }
 
-  &:hover {
-    .background {
-      filter: saturate(1.2) contrast(1.2) brightness(0.9);
+  @media (hover) {
```

Comment:
this is cool. Could you test it out on your iphone & on dektop?

Reply:
This looks slightly different from the other examples. Are you missing the colon and the second `hover`? i.e. `@media (hover: hover)`

Reply:
you're right, fixed it